bundles/ranvier-combat/commands/kill.js

```diff
@@ -16,26 +17,35 @@ module.exports = (srcPath) => {
         return Broadcast.sayAt(player, "You're too busy fighting!");
       }
 
-      const npc = Parser.parseDot(args, player.room.npcs);
-      if (!npc) {
+      const possibleTargets = [
+        ...player.room.npcs,
+        ...player.room.players
+      ];
+
+      const target = Parser.parseDot(args, possibleTargets);
+      if (!target) {
         return Broadcast.sayAt(player, "They aren't here.");
       }
 
-      if (npc.hasBehavior('pacifist')) {
-        return Broadcast.sayAt(player, `${npc.name} is a pacifist and will not fight you.`);
+      if (target.hasBehavior && target.hasBehavior('pacifist')) {
```

Comment:
Gotcha. I've already changed it locally to use `instanceof`. I'll take the isNpc thing into consideration, though. Might make the API nicer so we don't have to require Player or Npc everywhere just to do type checks.